Graphs/DynamicBarGraph/barGraph.test.js

```diff
@@ -0,0 +1,613 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { checkTicks, getDataAndConfig } from '../testHelper';
+import BarGraph from '.';
+
+const cheerio = require('cheerio');
+
+describe("Bar Graph", () => {
+    let config;
+    beforeAll(async () => {
+        config = await getDataAndConfig('DynamicBarGraph');
+    });
+
+    describe("Vertical Bar", () => {
+
+        describe("Numbered", () => {
+
+            let verticalNumber;
+            
+            beforeAll(async (done) => {
+                verticalNumber = await mount(
+                    <BarGraph
+                        width={500}
+                        height={500}
+                        configuration={config.verticalNumber}
+                        data={config.data}>
+                    </BarGraph>
+                );
+                setTimeout(() => { 
+                    verticalNumber.update(); 
+                    done(); 
+                }, 350); 
+
+            });
+           
+            it("Test svg", () => {
+                const svg = verticalNumber.find('svg').html();
+                const $ = cheerio.load(svg);
+                const svgHeight = $('svg').attr('height');
+                const svgWidth = $('svg').attr('width');
+                expect(svgHeight).toEqual("500");
+                expect(svgWidth).toEqual("500");
+            });
+
+            it("Total Bars", () => {
+                const graphBar = verticalNumber.find('.graph-bars').html();
+                const $ = cheerio.load(graphBar);
+                const noOfBars = $('.graph-bars').find('g').length;
+                expect(noOfBars).toBe(6);
+            });
+
+            it("Check xAxis ticks", () => {
```

Comment:
Must be like "xAxis ticks length" - Name must let you know, what are you trying to do. "Check" is bit general.